python_ta/patches/error_messages.py

```diff
@@ -4,18 +4,193 @@
 # global
 patch_data = {
     'pylint.checkers.base': {
+        'BasicChecker':
+            {'E0111': 'reversed() can only be called on sequence types. This '
+                      'typically occurs because you\'re trying to reverse '
+                      'something that isn\'t a str, list, or tuple.',
+             'W0101': 'Code after a return or raise statement will '
+                      'never be run, so either the function is '
+                      'returning early or you should remove it.',
+             'W0102': 'Dangerous default value %s as argument. '
+                      'Using mutable values as a default argument '
+                      'is a common cause of confusing and unwanted behaviour.',
+             'W0104': 'This statement doesn\'t have any effect, and could be '
+                      'removed without changing the behaviour of the program.',
+             'W0106': 'Expression "%s" isn\'t assigned to a variable.',
+             'W0108': 'The lambda function isn\'t necessary since the lambda '
```

Comment:
"...since you can pass the arguments directly to the function you're calling in the lambda's body."